docs/tutorial.rst

```diff
@@ -411,6 +411,23 @@ g) Quick access to file attributes of a UV* object (UVData, UVCal, UVBeam)
   >>> # Load object to instance name "uv" and will remain in interpreter
   pyuvdata_inspect.py -i <uv*_file>
 
+h) Update data arrays in place for UVData
```

Comment:
I think maybe this should be moved to between the existing d & e items, it should certainly come before the existing g item. 

Maybe we should also update the top-level text under this whole section ("UVData: Quick data access"), it's pretty terse at the moment.

Reply:
Done in bb9c09e